introduction.md

```diff
@@ -0,0 +1 @@
+Hi my name is Nate. I want to get good at Git. Ready be nice and learn me something.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.